src/libraries/System.Memory/src/System/Buffers/SequenceReader.cs

```diff
@@ -124,8 +124,8 @@ public SequenceReader(ReadOnlySequence<T> sequence)
         /// Peeks at the next value at specific offset without advancing the reader.
         /// </summary>
         /// <param name="offset">The offset from current position.</param>
-        /// <param name="value">The next value or default if at the end.</param>
-        /// <returns>False if at the end of the reader.</returns>
+        /// <param name="value">The next value, or the default value if at the end of the reader.</param>
+        /// <returns><c>true</c> if the reader is not at its end and the peek operation succeeded; <c>false</c> if at the end of the reader.</returns>`
```

Comment:
nit: Remove extra/unnecessary back-tick

```suggestion
        /// <returns><c>true</c> if the reader is not at its end and the peek operation succeeded; <c>false</c> if at the end of the reader.</returns>
```